busted/outputHandlers/json.lua

```diff
@@ -7,13 +7,23 @@ return function(options)
   local handler = require 'busted.outputHandlers.base'()
 
   handler.suiteEnd = function()
-    io_write(json.encode({
+    local error_info = {
       pendings = handler.pendings,
       successes = handler.successes,
       failures = handler.failures,
       errors = handler.errors,
       duration = handler.getDuration()
-    }))
+    }
+    local ok, result = pcall(function()
+      return json.encode(error_info)
+    end)
```

Comment:
The `pcall()` function will pass on extra arguments to the function referenced in the first argument, so you don't need an anonymous closure for this situation:

```suggestion
    local ok, result = pcall(json.encode, error_info)
```

Reply:
Ah yeah true.  Fixed